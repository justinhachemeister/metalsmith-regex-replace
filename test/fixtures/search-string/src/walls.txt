Teh walls in teh mall are like totally, totally tall.