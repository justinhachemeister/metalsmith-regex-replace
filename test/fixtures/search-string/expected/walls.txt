The walls in the mall are like totally, totally tall.